### PR TITLE
Add L0_multi_gpu test

### PR DESCRIPTION
**Pull request by Tabrizian**

### CODE REVIEW

File: qa/L0_multi_gpu/test.sh [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+        if [ $gpu_uuid != ${ananlyzer_gpus[$index]} ]; then
+            echo -e "\n***\n*** Model Analyzer is not using the correct GPUs.\n***"
+            return 1
+        fi
+        index=$((index+1))
+    done
+    return 0
+}
+
+function run_server_luanch_modes() {
```
Comment (xprotobeast2) [inline qa/L0_multi_gpu/test.sh:62]: should be `run_server_launch_modes`

Comment (xprotobeast2) [inline qa/L0_multi_gpu/test.sh:62]: Is this a refactor of L0_server_launch_modes/test.sh? If so, is there anyway we can move this and refactor over both files?

File: qa/L0_multi_gpu/test.sh [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+TRITON_SERVER_VERSION="20.11-py3"
+CLIENT_PROTOCOLS="http grpc"
+TEST_OUTPUT_NUM_ROWS=1
+TEST_OUTPUT_NUM_COLUMNS=7
+
+# Run the model-analyzer, both client protocols
+RET=0
+
+function check_gpus() {
+    ananlyzer_gpus=($@)
```
Comment (xprotobeast2) [inline qa/L0_multi_gpu/test.sh:42]: Should be `analyzer_gpus`

**Review by xprotobeast2 — CHANGES_REQUESTED** [review]

File: qa/L0_multi_gpu/test.sh (hunk 1 above)
Comment (Tabrizian) [reply qa/L0_multi_gpu/test.sh]: I put this test in the `L0_launch_modes` and removed `L0_multi_gpu`.

**Review by xprotobeast2 — APPROVED** [review]

**Review by dzier — APPROVED** [review]

**Pull request merged**